src/odemis/gui/comp/legend.py

```diff
@@ -60,18 +62,21 @@ def __init__(self, parent, wid=-1, pos=(0, 0), size=wx.DefaultSize,
 
         # Cannot be a constant because loading bitmaps only works after wx.App
         # has been created.
-        self._type_to_icon = (
-            (MD_AT_AR, img.getBitmap("icon/ico_blending_ang.png")),
-            (MD_AT_SPECTRUM, img.getBitmap("icon/ico_blending_spec.png")),
-            (MD_AT_EK, img.getBitmap("icon/ico_blending_spec.png")),
-            (MD_AT_EM, img.getBitmap("icon/ico_blending_sem.png")),
-            (MD_AT_OVV_TILES, img.getBitmap("icon/ico_blending_map.png")),
-            (MD_AT_OVV_FULL, img.getBitmap("icon/ico_blending_navcam.png")),
-            (MD_AT_HISTORY, img.getBitmap("icon/ico_blending_history.png")),
-            (MD_AT_CL, img.getBitmap("icon/ico_blending_opt.png")),
-            (MD_AT_FLUO, img.getBitmap("icon/ico_blending_opt.png")),
-            (MD_AT_SLIT, img.getBitmap("icon/ico_blending_slit.png")),
-        )
+        self._type_to_icon = {
+            MD_AT_AR: img.getBitmap("icon/ico_blending_ang.png"),
+            MD_AT_SPECTRUM: img.getBitmap("icon/ico_blending_spec.png"),
+            # MD_AT_TEMPORAL: img.getBitmap("icon/ico_blending_spec.png"),
+            MD_AT_TEMPSPECTRUM: img.getBitmap("icon/ico_blending_spec.png"),
+            # MD_AT_EK: img.getBitmap("icon/ico_blending_spec.png"),
+            MD_AT_EM: img.getBitmap("icon/ico_blending_sem.png"),
+            MD_AT_FIB: img.getBitmap("icon/ico_blending_fib.png"),
+            MD_AT_OVV_TILES: img.getBitmap("icon/ico_blending_map.png"),
+            MD_AT_OVV_FULL: img.getBitmap("icon/ico_blending_navcam.png"),
+            MD_AT_HISTORY: img.getBitmap("icon/ico_blending_history.png"),
+            MD_AT_CL: img.getBitmap("icon/ico_blending_opt.png"),
+            MD_AT_FLUO: img.getBitmap("icon/ico_blending_opt.png"),
+            MD_AT_SLIT: img.getBitmap("icon/ico_blending_slit.png"),
+        }
```

Comment:
For what reason are MD_AT_TEMPORAL and MD_AT_EK commented out?

Reply:
I don't have the right icons yet. Maxim will make them. (I've already indicated that in the commit description).

Reply:
I've now changed the file name for these icons to be the "real" file name when the file will exists... it's hopefully clearer that for now it's not going to work.